README.md

```
@@ -1,529 +1,529 @@
+import { ThemeProvider, dokanTheme, dokanDarkTheme } from "@wedevs/plugin-ui";
+
+<ThemeProvider pluginId="dokan" tokens={dokanTheme} darkTokens={dokanDarkTheme}>
+  <App />
+</ThemeProvider>
+```
+
+Available presets: `defaultTheme`, `dokanTheme`, `blueTheme`, `greenTheme`, `amberTheme`, `slateTheme` (and their `*DarkTheme` variants). Use `createTheme(tokens)` or `createDarkTheme(tokens)` to build a custom theme from a base.
+
+## Project Structure
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)